src/components/UpdateDB/index.js

```diff
@@ -0,0 +1,176 @@
+
+
+const fakeUpdates = {
+  lastUpdate: '16 Sep 2019, 13:94',
+  period: '1 Sep 2019 - 31 Oct 2019'
+}
+
+const priorUploadState = {
+  showPopUp: false,
+  ipcData: [],
+  uploading: false,
+  failedUpload: false,
+  successUpload: null
+}
+
+const UpdateDb = () => {
+  const [upload, setUpload] = useState(priorUploadState)
+
+  const loadIpcEntries = entries => {
+    setUpload({
+      showPopUp: true,
+      ipcData: entries,
+      uploading: false,
+      failedUpload: false,
+      successUpload: null
+    })
+  }
+
+  const cancelPopUp = () => {
+    setUpload({
+      showPopUp: false,
+      ipcData: [],
+      uploading: false,
+      failedUpload: false,
+      successUpload: null
+    })
+  }
+
+  const onYesContinue = () => {
+    setUpload({
+      showPopUp: false,
+      ipcData: upload.ipcData,
+      uploading: true,
+      failedUpload: false,
+      successUpload: null
+    })
+
+    const validateUpSert = res => {
+      console.log(res)
+      if (res.ok) {
+        res
+          .json()
+          .then(data => success(data))
+      } else {
+        return failed()
+      }
+    }
+
+    fetch('http://localhost:3001/donations/upload', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(upload.ipcData)
+    })
+      .then(validateUpSert)
+      .catch(err => console.log(err))
+  }
+
+  const failed = () => {
+    setUpload({
+      showPopUp: false,
+      ipcData: [],
+      uploading: false,
+      failedUpload: true,
+      successUpload: null
+    })
+  }
+  const success = data => {
+    setUpload({
+      showPopUp: false,
+      ipcData: [],
+      uploading: false,
+      failedUpload: false,
+      successUpload: data
+    })
+  }
+
+  return (
+    <div>
+      {upload.successUpload ? (
+        <SuccessUpload donorData={upload.successUpload} />
+      ) : (
+        <Box className="updatedb-box">
+          {upload.failedUpload ? (
+            <img
+              src={FailedImg}
+              alt="oops, an error occurred"
+              className="failimg"
+            />
+          ) : (
+            <img
+              src={UpdateDbImg}
+              alt="Update donor database"
+              className="uploadimg"
+            />
+          )}
+          <div className="updatedetails-container">
+            {upload.failedUpload ? <FailMsg /> : <UploadMsg />}
+            <FileHandlers loadIpcEntries={loadIpcEntries} CPU={cancelPopUp} />
+          </div>
+          {upload.showPopUp && (
+            <ConfirmUpload
+              CPU={cancelPopUp}
+              ipcEntries={upload.ipcData}
+              clickYes={onYesContinue}
+              success={success}
+              failed={failed}
+            />
+          )}
+          {upload.uploading && (
+            <ProgressBar onFailedUpload={failed} progress={upload.percentage} />
+          )}
+        </Box>
+      )}
+    </div>
+  )
+}
+
+const UploadMsg = () => {
+  return (
+    <div>
```

Comment:
Give this div a `display: flex`

Reply:
I added div with display:flex to the line above 101 in this image
![image](https://user-images.githubusercontent.com/55618945/74088624-54d4ab80-4ad3-11ea-9621-5b06eb53b3b7.png)